api/src/main/java/net/md_5/bungee/api/connection/ProxiedPlayer.java

```diff
@@ -237,6 +237,37 @@
      */
     void sendTitle(Title title);
 
+    /**
+     * Sends a resource pack to this player.
+     *
+     * @param url the url of resource pack
+     * @param hash the resource pack hash
+     * @throws IllegalArgumentException if hash is longer than 40 characters
+     */
+    void sendResourcePack(String url, String hash) throws IllegalArgumentException;
+
+    /**
+     * Gets the player's currently loaded resource pack hash.
+     *
+     * @return resource pack hash
+     */
+    String getResourcePackHash();
+
+    /**
+     * Gets the currently requested resource pack hash that is waiting for
```

Comment:
Same deal, and what would you even use this for?

Reply:
You're right, I don't see much use for exposing this to the API either and will remove it.